tests/components/home_plus_control/test_switch.py

```diff
@@ -434,7 +434,8 @@ async def test_update_with_api_error(
         },
     )
     for test_entity_uid in hass.data[DOMAIN]["home_plus_control_entry_id"][ENTITY_UIDS]:
```

Comment:
Don't access integration details in the tests. We should be able to know the uids since we're patching the library api.